sdk/pyproject.toml

```diff
@@ -31,7 +31,8 @@ classifiers = [
 ]
 dependencies = [
     "kubernetes>=27.2.0",
-    "jobset @ git+https://github.com/kubernetes-sigs/jobset.git@v0.7.3#subdirectory=sdk/python",
+    "pydantic>=2.10.0",
```

Comment:
This is required with the newer version of OpenAPI client: 
https://github.com/kubernetes-sigs/jobset/blob/main/sdk/python/jobset/api_client.py#L27